File: examples/chill/jacobi_simple.f
      SUBROUTINE jacobi(N,A)
      INTEGER N,T,I

      REAL*8 A(N,N)

      DO 10, T=2,100
         DO 20, I=2,N-1
            A(T,I)=A(T-1,I-1)+A(T-1,I)+A(T-1,I+1)
 20      CONTINUE
 10   CONTINUE

      END
      

File: examples/chill/testcases/jacobi_simple.c

#define N       1024


void jacobi(double A[N][N]) {
    int t,i;
    for(t = 1; t <= 100; t++) {
        for(i = 1; i < N-1; i++) {
            A[i][t] = A[i-1][t-1] + A[i][t-1] + A[i+1][t-1];
        }
    }
}


